demos/src/Examples/InteractivityComponentContent/React/index.spec.js

```diff
@@ -4,44 +4,47 @@ context('/src/Examples/InteractivityComponentContent/React/', () => {
   })
 
   it('should have a working tiptap instance', () => {
-    cy.get('.ProseMirror').then(([{ editor }]) => {
+    cy.get('.tiptap').then(([{ editor }]) => {
       // eslint-disable-next-line
       expect(editor).to.not.be.null
     })
   })
 
   it('should render a custom node', () => {
-    cy.get('.ProseMirror .react-component')
+    cy.get('.tiptap .react-component')
       .should('have.length', 1)
   })
 
   it('should allow text editing inside component', () => {
-    cy.get('.ProseMirror .react-component .content div')
+    cy.get('.tiptap .react-component .content div')
       .invoke('attr', 'contentEditable', true)
       .invoke('text', '')
       .type('Hello World!')
       .should('have.text', 'Hello World!')
   })
 
   it('should allow text editing inside component with markdown text', () => {
-    cy.get('.ProseMirror .react-component .content div')
+    cy.get('.tiptap .react-component .content div')
       .invoke('attr', 'contentEditable', true)
       .invoke('text', '')
-      .type('Hello World! This is **bold**.')
+      .click()
+    cy.get('.tiptap .react-component .content div')
+      .realType('Hello World! This is **bold**.')
+    cy.get('.tiptap .react-component .content div')
```

Comment:
Just so I understand better, does realType not support chaining or is this a timing thing?

Reply:
I'm no Cypress expert but:

- I initially tried with chaining, but it didn't work
- All our internal tests don't chain the `.real*` calls
- Examples on the [official repo](https://github.com/dmtrKovalenko/cypress-real-events) do use chaining
    - But like I said, I couldn't make it work...